internal/cmdliveinit/cmdliveinit.go

```diff
@@ -57,6 +69,7 @@ func NewRunner(ctx context.Context, factory cmdutil.Factory,
 	cmd.Flags().BoolVar(&r.Force, "force", false, "Set inventory values even if already set in Kptfile")
 	cmd.Flags().BoolVar(&r.Quiet, "quiet", false, "If true, do not print output message for initialization")
 	cmd.Flags().StringVar(&r.InventoryID, "inventory-id", "", "Inventory id for the package")
+	cmd.Flags().StringVar(&r.RGFile, "rg-file", rgfilev1alpha1.RGFileName, "ResourceGroup object filepath")
```

Comment:
suggest making flag changes in a single PR and make this UX change at last. so this feature won't block normal kpt release.

Reply:
I think we should come up with a more helpful help text here.

Reply:
Did we change anything here? This doesn't give any hints as to why I care about this flag. A suggestion would be to make it something like "Path where the file with the ResourceGroup resource will be created" (It isn't great, but at least provides a bit more context for what it does).

Reply:
So does the user provide a filename or a filepath here? If it is a filepath, we should make it clear whether we expect a relative or an absolute path. Currently it seems like we automatically assume the provided path is relative to the package root. It isn't obvious that this would be the behavior here.

Reply:
This is outdated code that got removed in the PR rework. We will not expose any CLI surface changes in this PR. A follow up PR later with surface changes, and new e2e tests will be done to enable this for our users.